src/utils/validCheck.ts

```
@@ -0,0 +1 @@
+export const isVaildRatingNumber = (rating: number) => !isNaN(rating) && rating >= 0 && rating <= 5;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)